From bc5ea53cb4dcc73f8f6207e901858d7686e47a0a Mon Sep 17 00:00:00 2001
From: Andrei Paulau <7134956@gmail.com>
Date: Sun, 8 Apr 2018 12:44:48 +0300
Subject: [PATCH] arm64/dts: add switch-delay for meson mali

Meson mali GPU operate in high clock frequency, need
this value be high to work in a stable state.

Signed-off-by: Andrei Paulau <7134956@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxbb.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxbb.dtsi b/arch/arm64/boot/dts/amlogic/meson-gxbb.dtsi
index 37b80e2..8ce10a4 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxbb.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-gxbb.dtsi
@@ -278,6 +278,7 @@
 			"pp2", "ppmmu2";
 		clocks = <&clkc CLKID_CLK81>, <&clkc CLKID_MALI>;
 		clock-names = "bus", "core";
+		switch-delay = <0xffff>;
 
 		/*
 		 * Mali clocking is provided by two identical clock paths
